atlas-fastq-provider-functions.sh

```diff
@@ -1180,6 +1181,80 @@ fetch_library_files_from_ena() {
         done 
 
     fi
+
+    # A sleep here to try to deal with cases where we get a success exit code
+    # above, but the file does not exist when checked below. Suspect some sort of
+    # file system latency, so maybe if we wait a bit before checking it will solve
+    # the issue.
+
+    sleep 10
+
+    localFastqPath=${ISL_RAW_DIR}/$(get_library_path $library)
```

Comment:
You need to change any ISL* references, atlas-fastq-provider should be standalone

Reply:
Plus get_library_path etc are not relevant here. You need to change the checks logic such that it refers to the $outputDir etc above.